packages/strapi-admin/admin/src/components/Logout/index.js

```diff
@@ -0,0 +1,58 @@
+/**
+*
+* Logout
+*
+*/
+
+import React from 'react';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import { ButtonDropdown, DropdownItem, DropdownMenu, DropdownToggle } from 'reactstrap';
+import auth from 'utils/auth';
+
+import styles from './styles.scss';
+
+class Logout extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  state = { isOpen: false };
+
+  handleGoTo = () => {
+    const id = get(auth.getUserInfo(), 'id') || get(auth.getUserInfo(), '_id');
+    this.context.router.history.push(`/plugins/content-manager/user/${id}?redirectUrl=/plugins/content-manager/user/?page=0&limit=0&sort=id&source=users-permissions`);
```

Comment:
Don't put a slash after the word `user`. So `/user/?page=...` becomes `/user?page=`.

Reply:
Also, I prefer this syntax.

```js
this.context.router.history.push({
  pathname: `/plugins/content-manager/user/${id}?redirectUrl=/plugins/content-manager/user`,
  search: `?page=0&limit=0&sort=id&source=users-permissions`
});
```
It helps to distinguish the URL and the parameters. It also harmonizes the codebase because we are using the object syntax in the Content Manager plugin.